docs/en/administration/management/Backup_and_restore.md

```diff
@@ -189,9 +189,11 @@ You can optimize the performance of BACKUP or RESTORE jobs by modifying the foll
 
 | Configuration item      | Description                                                                             |
 | ----------------------- | -------------------------------------------------------------------------------- |
-| upload_worker_count     | The maximum number of threads for the upload tasks of BACKUP jobs on a BE node. Default: `1`. Increase the value of this configuration item to increase the concurrency of the upload task. |
-| download_worker_count   | The maximum number of threads for the download tasks of RESTORE jobs on a BE node. Default: `1`. Increase the value of this configuration item to increase the concurrency of the download task. |
-| max_download_speed_kbps | The upper limit of the download speed on a BE node. Default: `50000`. Unit: KB/s. Usually, the speed of the download tasks in RESTORE jobs will not exceed the default value. If this configuration is limiting the performance of RESTORE jobs, you can increase it according to your bandwidth.|
+| make_snapshot_worker_count     | The maximum number of threads for the make snapshot tasks of BACKUP jobs on a BE node. Default: `5`. Increase the value of this configuration item to increase the concurrency of the make snapshot task. |
+| release_snapshot_worker_count     | The maximum number of threads for the release snapshot tasks of failed BACKUP jobs on a BE node. Default: `5`. Increase the value of this configuration item to increase the concurrency of the release snapshot task. |
+| upload_worker_count     | The maximum number of threads for the upload tasks of BACKUP jobs on a BE node. Default: `0`. `0` indicates setting the value to the number of CPU cores on the machine where the BE resides. Increase the value of this configuration item to increase the concurrency of the upload task. |
+| download_worker_count   | The maximum number of threads for the download tasks of RESTORE jobs on a BE node. Default: `0`. `0` indicates setting the value to the number of CPU cores on the machine where the BE resides. Increase the value of this configuration item to increase the concurrency of the download task. |
```

Comment:
download_worker_count and move_dir_worker_count are both for `RESTORE` jobs, is it possible to combine them together, so user just need to tune one parameter for the restore job? It may be hard to figure out the ratio between download and movedir thread number.

Reply:
both use download_worker_count ?

Reply:
share the same configvar or have a fixed ratio between download and move, e.g. movedir thread number = half of the download worker count.

Reply:
shared the same configvar